source/elements/oneDAL/include/oneapi/dal/common.hpp

```diff
@@ -0,0 +1,58 @@
+namespace oneapi::dal {
+
```

Comment:
should this file include cstdint/other related headers?

Reply:
It actually includes, but I dropped this includes since they didn't make any sense so far for generating docs.